### PR TITLE
fix(input): revert experimental inputs back to 40px, remove hover styles

### PR DESCRIPTION
**Pull request by tw15egan**
Closes https://github.com/IBM/carbon-components/issues/1424

Updates the following form items:
- Date Picker
- Dropdown
- List Box
- Number Input
- Select
- Slider
- Text Input
- Time Picker

I did not touch TextArea. @jnm2377, I would just include these updates in your TextArea PR

#### Changelog

**New**

- ~Added `'IBM Plex Sans'` to `$font-family-sans-serif`, as italic styles were not rendering properly without it.~ After talking with @asudoh, a change like this should be discussed in more detail in a separate PR 
- `bx--form__helper-text` now uses italics

**Changed**

- All inputs are now 40px by default
- Optional text now comes directly after the label
- The positioning of arrows/chevrons due to new sizes 

**Removed**

- Hover styles from input elements

#### Testing / Reviewing

[Staging Link](http://carbon-dev-environment-field-update-fluent-gelada.mybluemix.net/?nav=text-input)

### CODE REVIEW

File: src/globals/scss/_typography.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import 'vars';
 
 $font-family-mono: 'ibm-plex-mono', 'Menlo', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', Courier, monospace !default;
-$font-family-sans-serif: 'ibm-plex-sans', Helvetica Neue, Arial, sans-serif !default;
+$font-family-sans-serif: 'IBM Plex Sans', 'ibm-plex-sans', Helvetica Neue, Arial, sans-serif !default;
```
Comment (asudoh) [inline src/globals/scss/_typography.scss]: Wanted to double-check if this is a temporary change, or we really had an issue with fonts.

File: src/components/text-input/text-input.hbs [hunk 2]
```diff
@@ -42,6 +42,9 @@
 
 <div {{#if password}} data-text-input {{/if}} class="bx--form-item bx--text-input-wrapper">
   <label for="text-input-5" class="bx--label">Text Input label</label>
+  <div class="bx--form__helper-text">
```
Comment (asudoh) [inline src/components/text-input/text-input.hbs:45]: Wanted to double-check if the markup change works for stable mode (or stable mode was like this before).

**Review by asudoh — COMMENTED** [review]
LGTM basically, just a couple of comments - Thanks @tw15egan!

File: src/globals/scss/_typography.scss (hunk 1 above)
Comment (tw15egan) [reply src/globals/scss/_typography.scss]: Italic fonts were not rendering for me unless I added this in 🤷‍♂️

**Review by jnm2377 — APPROVED** [review]
Looks good to me! Thanks for the tag, I'll make sure to add these changes to Text Area.

**Comment by jeanservaas** [comment]
This looks good and I'd still merge it so we can talk about it at component review... there is a weird icon in the password input though, and weird spacing...

image

**Comment by alisonjoseph** [comment]
@jeanservaas the password input I don't think has been designed for experimental. We aren't showing it on the website, so on worries about that right now. 👍

**Review by alisonjoseph — APPROVED** [review]
LGTM, other than the same question about the font change.

File: src/components/text-input/text-input.hbs (hunk 2 above)
Comment (alisonjoseph) [reply src/components/text-input/text-input.hbs:45]: Double checked and seems ok 👍

**Pull request merged**